crates/dkg-cli/Cargo.toml

```diff
@@ -1,15 +1,21 @@
 [package]
 name = "dkg-cli"
-version = "0.1.0"
-authors = ["Georgios Konstantopoulos <me@gakonst.com>"]
-edition = "2018"
+version = "0.3.0"
+authors = ["Georgios Konstantopoulos <me@gakonst.com>", "Michael Straka <mstraka100@gmail.com>"]
+edition = "2022"
+build = "build.rs"
+
+[[bin]]
```

Comment:
It seems I'm unable to (after changing the edition to 2021) `cargo build` this project it seems the problem is in `build.rs`

```
error: failed to run custom build command for `dkg-cli v0.3.0 (/Users/piersy/projects/celo-threshold-bls-rs/crates/dkg-cli)`

Caused by:
  process didn't exit successfully: `/Users/piersy/projects/celo-threshold-bls-rs/target/debug/build/dkg-cli-49e95d0c07e2951c/build-script-build` (exit status: 101)
  --- stdout
  cargo:rerun-if-changed=../../solidity/

  --- stderr
  thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', crates/dkg-cli/build.rs:19:61
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

I also noticed that `Cargo.lock` was updated by the build, which feels like it should not happen, but could be a consequence of changing the edition in `dkg-cli`

I dug into the errors a bit and it seems that compilation can fail but the result is still an ok result, the compiler output needs to be introspected in greater depth to see if there were errors. When I did get the build script to run  I then faced a host more errors inside the `dkg-cli` crate.

I tried to push my bracnh up but I don't have permissions right now.

Reply:
I had the same, the mapping seems to require a full path in the config.

For the new errors, it seems like [this file](https://github.com/celo-org/celo-threshold-bls-rs/pull/108/files#diff-51a347bc227b1033b7aa387d6a55c3e63beae636926c9fd93e40a02183d2d210) was deleted and is missing now. Bringing it back fixes the build for me.